src/components/Form/DocumentSelection.jsx

```diff
@@ -30,7 +30,7 @@ const DocumentSelection = ({ htmlForAndIdProp, handleDocType, docType }) => (
         name="document"
       >
         {Object.entries(DOC_TYPES).map(([key, value]) => (
-          <MenuItem key={key} value={key}>
+          <MenuItem key={key} value={value}>
```

Comment:
I admit the naming here is confusing, but the code itself is correct. The "key" here is the data value, while the "value" is the user-friendly form of that value. The value was not intended to be saved into data, or used in code, which is why it looks this way.

Reply:
The data was being stored into the TTL file as "BankStatement" and "DriversLicense" for the TTL file identifier and additionalType instead of "Bank Statement" and "Driver's License", so I thought it was a bug.

Reply:
That's actually the intent of this code, not a bug. Store the data in a way that's easy for machines to interpret, display it in a way that's easy for humans to interpret. Spaces like we have in "Bank Statement" or "Driver's License" make the strings harder to parse in code. The goal is to store these in a format that makes the data easier to query and reference in the future.

Reply:
Alright, that should be fine. I'll close this PR.